CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ## 20.41.0 - 2024-03-25
 
+### PaymentSheet
+* [Added][8148](https://github.com/stripe/stripe-android/pull/8148) Added `intentMode` & `setupToFutureUsage` fields to Link popup request to update `Link` confirm button based on intent information.
```

Comment:
I don't think we need this one, it's not _really_ user facing. If anything we'd want to reword it for something along the lines of changing the pay button text.